app/src/main/java/com/example/bottomnav/Parse.java

```diff
@@ -19,12 +19,12 @@
 import java.util.regex.Pattern;
 
 public class Parse {
-    // Const name -> ConstContents
-    public HashMap<String, ConstContents> constRepo = new HashMap<>();
-    // Struct name -> StructContents
-    public HashMap<String, ArrayList<StructContents>> structRepo = new HashMap<>();
-    // CAN ID -> CAN Struct
-    public HashMap<String, String> IDToStruct = new HashMap<>();
+    private HashMap<String, ConstContents> constRepo = new HashMap<>();
```

Comment:
This is a lot of stuff to store, at some point we should revisit this to determine if we can reduce this down